src/filelock/_api.py

```diff
@@ -164,7 +164,7 @@ def acquire(
 
         lock_id = id(self)
         lock_filename = self._lock_file
-        start_time = time.monotonic()
+        start_time = time.perf_counter()  # noqa: SC200
```

Comment:
Remove these noqa

Reply:
Should add the word to https://github.com/tox-dev/py-filelock/blob/main/whitelist.txt#L3 instead

Reply:
what word?

Reply:
What word you added this noqa for.

Reply:
done, I guess